tools/stern/src/Stern/API.hs

```diff
@@ -78,7 +78,7 @@ start :: Opts -> IO ()
 start o = do
   e <- newEnv o
   s <- Server.newSettings (server e)
-  runSettings s (pipeline e)
+  Server.runSettingsWithShutdown s (pipeline e) Nothing `finally` pure () -- nothing to clean up?
```

Comment:
```suggestion
  Server.runSettingsWithShutdown s (pipeline e) Nothing
```

Nit: Why write that `finally`? Stern probably has nothing important running, so it makes sense that it doesn't have anything to cleanup.

Reply:
wasn't sure about this, just thought sometimes it's nice to be explicit about doing nothing.  makes it clear that it's not an oversight, but intentional.  maybe here it's just noise, though.